### ISSUE TITLE
--force option is invalid, but knife client delete tells you to use it

### ISSUE THREAD

**Issue by jkeiser** [chef/chef#1652]
```
jkeiser@johns-mbp-3 ~/o/c/o/c/playground> knife client delete chef-validator
Do you really want to delete chef-validator? (Y/N)Y
FATAL: You must specify --force to delete the validator client chef-validator
jkeiser@johns-mbp-3 ~/o/c/o/c/playground> knife client delete chef-validator --force
Error: invalid option: --force
```

My version is master (though I imagine this happens on other versions):

```
jkeiser@johns-mbp-3 ~/o/c/o/c/playground> knife --version
Chef: 11.16.0.alpha.0
```

**Comment by dpnl87** [comment]
@stevendanna Am I correct this can be closed with the merge of #1667

**Comment by stevendanna** [comment]
@dpnl87 yup!
